templates/weblogic.service.j2

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Weblogic Service for nDelius
+After=network.target
+
+[Service]
+Type=simple
+{% if wls_service_user is defined and wls_service_user.name is defined %}
+User={{ wls_service_user.name }}
+{% else %}
+User=oracle
+{% endif %}
+WorkingDirectory={{ mw_home | default('/u01/app/oracle/middleware') }}/user_projects/domains/NDelius
+ExecStart={{ mw_home | default('/u01/app/oracle/middleware') }}/user_projects/domains/NDelius/startWebLogic.sh
+Restart=on-failure
+ExecStop={{ mw_home | default('/u01/app/oracle/middleware') }}/user_projects/domains/NDelius/bin/stopWebLogic.sh
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
missing new line

Reply:
Sorry, where is a new line missing? ExecStop is all one line, concatenating a string.

Reply:
If it's a final blank line, i'll add one in. I dont think the systemd build on centos needs a final blank line; some arch builds do.